src/Microsoft.AspNetCore.Server.HttpSys/MessagePump.cs

```diff
@@ -32,7 +33,7 @@ internal class MessagePump : IServer
 
         private readonly ServerAddressesFeature _serverAddresses;
 
-        public MessagePump(IOptions<HttpSysOptions> options, ILoggerFactory loggerFactory)
```

Comment:
Probably need to factor the new auth stuff into the 'core stuff' (and the security base class stuff).  Hosting should add the core stuff so everyone can add schemes...